### ISSUE TITLE
  How to get   mask_loss 

### ISSUE THREAD

**Issue by donghang941114** [STVIR/pysot#255]
if cfg.MASK.MASK:
            # TODO
            mask, self.mask_corr_feature = self.mask_head(zf, xf)
            mask_loss = None
            outputs['total_loss'] += cfg.TRAIN.MASK_WEIGHT * mask_loss
            outputs['mask_loss'] = mask_loss
        return outputs

**Comment by yc-zc** [comment]
I am having troubles understanding your issues. 
Could you provide more details?

**Comment by yc-zc** [comment]
Since no further question is asked, I'm closing it for now.
Please reopen it or create a new issue if you have further question.
